packages/mirage-conduit/mirage-conduit.2.2.0/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+maintainer:   "anil@recoil.org"
+authors:      ["Anil Madhavapeddy" "Thomas Leonard" "Thomas Gazagnaire"]
+homepage:     "https://github.com/mirage/ocaml-conduit"
+dev-repo:     "https://github.com/mirage/ocaml-conduit.git"
+bug-reports:  "https://github.com/mirage/ocaml-conduit/issues"
+tags:         "org:mirage"
+
+build:   ["ocamlfind" "query" "conduit.mirage"]
+depends: [
+  "mirage-types" {>= "2.0.0"}
+  "mirage-dns" {>= "2.0.0"}
+  "tcpip"
```

Comment:
good idea, I'll add a constraint to use the latest tcpip release (with all the nice fixes)